descriptions/ab+/spa.lua

```diff
@@ -741,103 +741,103 @@ EID.descriptions[languageCode].trinkets={
 	{"9", "Gusano de Pulso", "Las lágrimas aumentan y disminuyen su tamaño#Afecta a la hitbox de la lágrima"}, -- Pulse Worm
-	{"10", "Gusano Ondulante", "↑ {{Tears}} Lágrimas +0.3#Las lágrimas se mueven en ondas"}, -- Wiggle Worm
+	{"10", "Gusano Ondulante", "↑ {{Tears}} Lágrimas +0.4#Las lágrimas se mueven en ondas"}, -- Wiggle Worm
 	{"11", "Gusano Circular", "Las lágrimas se mueven en espiral a gran velocidad"}, -- Ring Worm
 	{"12", "Gusano plano", "Lágrimas 50% más grandes#Aumenta el retroceso"}, -- Flat Worm
 	{"13", "Tarjeta de crédito", "{{Shop}} Permite llevarse un objeto de la tienda gratis"}, -- Credit Card
-	{"14", "Callo", "Otorga inmunidad a las espinas y el fluido"}, -- Callus
+	{"14", "Callo", "Otorga inmunidad a las espinas de suelo y el fluido"}, -- Callus
 	{"15", "Roca de la suerte", "{{Coin}} Genera monedas al destruir rocas"}, -- Lucky Rock
 	{"16", "Uña del pie de Mamá", "El pie de mamá cae aleatoriamente en la habitación cada 60 segundos"}, -- Mom's Toenail
 	{"17", "Pintalabios negro", "{{BlackHeart}} +5% de probabilidad de reemplazar los corazones de alma por corazones negros"}, -- Black Lipstick
 	{"18", "Folleto Biblico", "{{EternalHeart}} +3% de probabilidad de encontrar corazones eternos"}, -- Bible Tract
 	{"19", "Clip de Papel", "{{GoldenChest}} Permite abrir los cofres dorados sin usar llaves"}, -- Paper Clip
 	{"20", "Pata de mono", "{{BlackHeart}} Al tener medio corazón, deja caer un corazón negro#{{Danger}} Desaparece luego de 3 usos"}, -- Monkey Paw
 	{"21", "Papel misterioso", "Otorga aleatoriamente el efecto de:#{{Collectible327}} La Polaroid#{{Collectible328}} El Negativo#{{Trinket48}} Una página perdida#{{Trinket23}} Cartel de desaparición"}, -- Mysterious Paper
-	{"22", "Cola de Demonio", "{{Heart}} Disminuye a 20% la probabilidad de generar corazones rojos#{{BlackHeart}} Convierte los corazones rojos en corazones negros#{{Key}} Aumenta la probabilidad de generar llaves"}, -- Daemon's Tail
-	{"23", "Cartel de Desaparecido", "{{SacrificeRoom}} Reaparecerás como {{Player10}} El Perdido al morir"}, -- Missing Poster
-	{"24", "Moneda de Trasero", "{{Coin}} +20% de probabilidad de generar monedas al romper caca#Recoger monedas hará que te tires pedos"}, -- Butt Penny
-	{"25", "Dulce misterioso", "Te tiras un pedo o generas una caca cada 30 segundos"}, -- Mysterious Candy
+	{"22", "Cola de Demonio", "{{Heart}} Disminuye a 20% la probabilidad de generar corazones#{{BlackHeart}} Convierte los corazones en corazones negros#{{Key}} Aumenta la probabilidad de generar llaves"}, -- Daemon's Tail
+	{"23", "Cartel de Desaparecido", "{{SacrificeRoom}} Reaparecerás como {{Player10}} El Perdido al morir en una sala de Sacrificio#También puede usarse en otras salas una vez desbloqueado {{Player10}} El Perdido"}, -- Missing Poster
```

Comment:
Line 1274: Info about sacrifice room is displayed if player hasn't unlocked The Lost

```suggestion
	{"23", "Cartel de Desaparecido", "{{Player10}} Reaparecerás como El Perdido al morir"}, -- Missing Poster
```
{{SacrificeRoom}} Shouldn't be there, I forgot to upload the corrected description